cypress/integration/promiseFunction.spec.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+Cypress.Commands.add('checkNotificationExists', (promise, element, id) => {
+  cy.contains(`It was ${promise} on the element: ${element}, id: ${id}.`)
+    .should('be.visible');
+});
+
+describe('Promise function app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it(`should resolve 'click' promise for 'login' field`, () => {
+    cy.get('#login').click();
+    cy.checkNotificationExists('click', 'INPUT', 'login');
+  });
+
+  it(`should resolve 'input' promise for 'login' field`, () => {
+    cy.get('#login').type('login');
+    cy.checkNotificationExists('input', 'INPUT', 'login');
+  });
+
+  it(`should resolve 'blur' promise for 'login' field`, () => {
+    cy.get('#login').click();
+    cy.get('body').click({ force: true });
```

Comment:
why not https://docs.cypress.io/api/commands/blur.html#Usage ?

Reply:
done!